src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -473,7 +473,7 @@ public static void WriteIntPtr(IntPtr ptr, int ofs, IntPtr val)
 #if TARGET_64BIT
             WriteInt64(ptr, ofs, (long)val);
 #else // 32
-#pragma warning disable CA2020 // Prevent from behavioral change
+#pragma warning disable CA2020 // Prevent behavioral change
```

Comment:
Can we get rid of these pragma warning disable instead? E.g. `WriteInt32(ptr, ofs, (int)(nint)val);`?

Reply:
Can we change the method signature to:
```cs
public static void WriteIntPtr(IntPtr ptr, int ofs, nint val)
```

Reply:
I did this for all `Marshal` methods where appropriate: https://github.com/dotnet/runtime/pull/118335

Reply:
And `Interlocked`: https://github.com/dotnet/runtime/pull/118336